Apps/Src/SendTritium.c

```diff
@@ -2,693 +2,777 @@
  * @copyright Copyright (c) 2018-2023 UT Longhorn Racing Solar
  * @file SendTritium.c
  * @brief Function implementations for the SendTritium application.
- * 
- * This contains functions relevant to updating the velocity and current setpoitns
- * of the Tritium motor controller. The implementation includes a normal current
- * controlled mode, a one-pedal driving mode (with regenerative braking), and cruise control.
- * The logic is determined through a finite state machine implementation.
- * 
- * If the macro SENDTRITIUM_EXPOSE_VARS is defined prior to including SendTritium.h, 
- * relevant setters will be exposed as externs for unit testing
- * and hardware inputs won't be read and motor commands won't be sent over MotorCAN.
- * If the macro SENDTRITIUM_PRINT_MES is also defined prior to including SendTritium.h,
- * debug info will be printed via UART.
+ *
+ * This contains functions relevant to updating the velocity and current
+ * setpoitns of the Tritium motor controller. The implementation includes a
+ * normal current controlled mode, a one-pedal driving mode (with regenerative
+ * braking), and cruise control. The logic is determined through a finite state
+ * machine implementation.
+ *
+ * If the macro SENDTRITIUM_EXPOSE_VARS is defined prior to including
+ * SendTritium.h, relevant setters will be exposed as externs for unit testing
```

Comment:
```suggestion
 * SendTritium.h, relevant setters will be exposed for unit testing
```
(Since we're no longer using externs)